examples/nginx-grafana/README.md

```diff
@@ -246,8 +244,85 @@ Query id: 11d071e4-6028-47da-826d-9db42176f8e5
+Attaching to nginx-grafana-access_log-1, nginx-grafana-grafana-1, nginx-grafana-proton-1
+nginx-grafana-proton-1      | Processing configuration file '/etc/proton-server/config.yaml'.
+nginx-grafana-proton-1      | Merging configuration file '/etc/proton-server/config.d/data-paths.xml'.
+nginx-grafana-proton-1      | Merging configuration file '/etc/proton-server/config.d/docker_related_config.xml'.
+nginx-grafana-proton-1      | Merging configuration file '/etc/proton-server/config.d/logger.xml'.
+nginx-grafana-proton-1      | Merging configuration file '/etc/proton-server/config.d/openssl.xml'.
+nginx-grafana-proton-1      | Merging configuration file '/etc/proton-server/config.d/user-directories.xml'.
+nginx-grafana-proton-1      | Logging information to /var/log/proton-server/proton-server.log
+nginx-grafana-proton-1      | Logging errors to /var/log/proton-server/proton-server.err.log
+nginx-grafana-access_log-1  | Using the following config for IPinfo:
+nginx-grafana-access_log-1  | {"cache_enabled":true,"token":"xxxxxxxxxxxxxx","open_browser":true}
+nginx-grafana-access_log-1  | Extracting only IP addresses from the 1st column in /tmp/nginx_export.csv to /tmp/nginx_export.ipinfo.tmp ...
+nginx-grafana-access_log-1  | Removing duplicate IP addresses from /tmp/nginx_export.ipinfo.tmp ...
+nginx-grafana-access_log-1  | Combined all IP addresses into a single file: /tmp/nginx_export.ipinfo.
+nginx-grafana-access_log-1  | removed '/tmp/nginx_export.ipinfo.tmp'
+nginx-grafana-access_log-1  | Extracting the rest of the data from /tmp/nginx_export.csv to /tmp/nginx_import.csv ...
+nginx-grafana-access_log-1  | renamed '/tmp/nginx_export.csv' -> '/tmp/nginx_import.csv'
+nginx-grafana-access_log-1  | Geo-locating all the IP addresses in bulk using the IPInfo API (https://ipinfo.io):
+nginx-grafana-access_log-1  |    Total IP addresses that will be looked up in bulk using /tmp/nginx_export.ipinfo: 10000.
+nginx-grafana-access_log-1  |    Geo-lookup of 10000 IP addresses written to file: /tmp/nginx_import.ipinfo.csv.
+nginx-grafana-access_log-1  |    Complete!
+nginx-grafana-access_log-1 exited with code 0
+nginx-grafana-grafana-1     | ✔ Downloaded and extracted timeplus-proton-datasource v1.0.3 zip successfully to /var/lib/grafana/plugins/timeplus-proton-datasource
+nginx-grafana-grafana-1     | 
+nginx-grafana-grafana-1     | Please restart Grafana after installing or removing plugins. Refer to Grafana documentation for instructions if necessary.
+nginx-grafana-grafana-1     | 
+nginx-grafana-grafana-1     | logger=settings t=2024-07-19T21:43:50.364886485Z level=info msg="Starting Grafana" version=11.1.0 commit=5b85c4c2fcf5d32d4f68aaef345c53096359b2f1 branch=HEAD compiled=2024-07-19T21:43:50Z
+...
+``` 
+
+5. Open http://localhost:3000 in your web browser:
+
+
+6. Click on the hamburger menu at the top left and click on "Data sources" to navigate to the "Data sources" page so you can setup a new data source:
```

Comment:
Reduce wordiness, and change "setup" to "set up"

Click on the hamburger menu at the top left and click on "Data sources" to set up a new data source on that page:

Reply:
It's no longer there. Seems like you are reviewing an old version of this file?